### DISCUSSION TITLE
How to show all the available options? and not all minus the selected ones

### DISCUSSION THREAD

**Discussion by sedghi** [JedWatson/react-select#5745] [Q&A]
Right now i now I see 

![CleanShot 2023-09-01 at 16 29 41](https://github.com/JedWatson/react-select/assets/7490180/3e85422b-c1c7-4407-ad86-bedab8708f9d)

I would like however, to show all the presets (options) in the dropdown and not all minus the selected one

**Comment by manjushsh** [comment]
Hi @sedghi, as mentioned in [documentation](https://react-select.com/props) you can show all options by passing `hideSelectedOptions={false}` to Select

Example on [Code Sandbox](https://codesandbox.io/s/react-select-input-in-menulist-forked-tqhxgt?file=/hideSelected.js)

**Comment by sedghi** [comment]
Thanks a lot!
